Sources/Rownd/Models/AppleSignUpCoordinator.swift

```diff
@@ -157,11 +157,13 @@ class AppleSignUpCoordinator: NSObject, ASAuthorizationControllerDelegate, ASAut
                         }
                     } catch ApiError.generic(let errorInfo) {
                         if errorInfo.code == "E_SIGN_IN_USER_NOT_FOUND" {
-                            Rownd.requestSignIn(jsFnOptions: RowndSignInJsOptions(
-                                token: idToken,
-                                loginStep: .noAccount,
-                                intent: .signIn
-                            ))
+                            Task { @MainActor in
```

Comment:
I actually went down this route, but it can cause some interesting compilation errors across a codebase that needs to call it.

That said, I think I actually solved the need to use `@MainActor` or the main dispatch queue when calling `requestSignIn()` by handling all of that within the code that actually displays the hub. So this particular code (and similar usages) could probably be moved outside of a `Task`.